_layouts/open-data-solutions.html

```diff
@@ -0,0 +1,1632 @@
+                                                        When a natural disaster or disease outbreak occurs there is a rush to establish accurate health care location data that can be used to support people on the ground.
+                                                        <br><br> 
+                                                        <p>This has been demonstrated by events such as the Haiti earthquake and the Ebola epidemic in West Africa. As a result valuable time is wasted establishing accurate and accessible baseline data.
+                                                          Healthsites.io establishes this data and the tools necessary to upload, manage and make the data easily accessible.</p>                                       
+                                                        <a style="font-style: italic;">(source: website)</a>
+                                                      </p>
+                                                      <p>
+                                                        <br>
+                                                        <p style="color: black; padding-bottom: 3%;"><strong>Features</strong></p>
+                                                        <ul style="list-style-type: square;">
+                                                          <li><strong>Data model: </strong> Each healthsite is stored as a geometry object in the database.<br></li>
+                                                          <li><strong>Share healthsite data: </strong>Please contact us should you wish to share data and we will assist you.<br></li>
+                                                        </ul>
+                                                      </p>
+                                                      <div class="block-options">
+                                                        <ul style="display: inline-flex; padding-top: 5%;">
+                                                          {% for tool in page['Healthsites'].Tools %}
+                                                            <li><a class="external" href="{{ tool.URL }}" style="color: black;">{{ tool.Name }}</a></li>
+                                                          {% endfor %}
+                                                        </ul>
+                                                      </div>
+                                                    </div>
+                                                </div>  
+
+                                           
+                                                <div class="p-6 rounded-lg w-full" id="mapbox" role="tabpanel" aria-labelledby="profile-tab">
+                                                  <h3 class="text-lg font-bold mb-2" style="color: #d63f3f; font-family: Barlow Condensed, sans-serif; font-weight: bold; text-transform: uppercase; letter-spacing: .045em;">Mapbox</h3>
```

Comment:
Should mapbox be listed here ?

Reply:
no, they should not you are right -- deleted